docs/development/module/fields.md

```diff
@@ -252,6 +334,7 @@ label: Ear tag
 bundles:
   - animal
 ```
+Note that the file name is in the form `farm_flag.flag.ear_tag.[id].yml`
```

Comment:
This should be `farm_id_tag.id_tag.[id].yml`

And I see that there is a typo in the existing docs (which you copied from, so my fault!) - can you fix that too while you're at it?

`animal/config/install/farm_flag.flag.ear_tag.yml` should be `animal/config/install/farm_id_tag.id_tag.ear_tag.yml`